src/main/java/org/apache/hive/hcatalog/data/JsonSerDe.java

```diff
@@ -106,26 +109,23 @@ public void initialize(Configuration conf, Properties tbl)
     List<TypeInfo> columnTypes;
     StructTypeInfo rowTypeInfo;
 
-
-    LOG.debug("Initializing JsonSerDe");
-    LOG.debug("props to serde: {}", tbl.entrySet());
-
+    LOG.debug("Initializing JsonSerDe: {}", tbl.entrySet());
 
     // Get column names and types
     String columnNameProperty = tbl.getProperty(serdeConstants.LIST_COLUMNS);
     String columnTypeProperty = tbl.getProperty(serdeConstants.LIST_COLUMN_TYPES);
     final String columnNameDelimiter = tbl.containsKey(serdeConstants.COLUMN_NAME_DELIMITER) ? tbl
         .getProperty(serdeConstants.COLUMN_NAME_DELIMITER) : String.valueOf(SerDeUtils.COMMA);
     // all table column names
-    if (columnNameProperty.length() == 0) {
-      columnNames = new ArrayList<String>();
+    if (columnNameProperty.isEmpty()) {
+      columnNames = Collections.emptyList();
```

Comment:
any other copied classes need update?

Reply:
Yes, I verified the others (most are shims and not copied classes)